src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -567,9 +567,10 @@ export const Deposit: React.FC<DepositProps> = ({
     browserHistory.goBack()
   }, [browserHistory])
 
-  const validateCryptoAmount = useCallback(
+  const _validateCryptoAmount = useCallback(
     async (value: string) => {
-      if (!accountId) return
+      if (!accountId || !contextDispatch) return
+      if (state?.loading) return
```

Comment:
For deposits, noticed this allowed us to avoid overfetching and make things longer.
Unfortunately adding this for withdraws borked them, and didn't get to proper validation there with debounce either.